### PR TITLE
Add Xumm provider

### PR DESCRIPTION
**Pull request by DominiqueBlomsma**
Add Xumm provider

### CODE REVIEW

File: src/AspNet.Security.OAuth.Xumm/AspNet.Security.OAuth.Xumm.csproj [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(DefaultNetCoreTargetFramework)</TargetFrameworks>
+    <PackageValidationBaselineVersion>6.0.9</PackageValidationBaselineVersion>
```
Comment (martincostello) [inline src/AspNet.Security.OAuth.Xumm/AspNet.Security.OAuth.Xumm.csproj]:
```suggestion
    <PackageValidationBaselineVersion>6.0.11</PackageValidationBaselineVersion>
```

File: test/AspNet.Security.OAuth.Providers.Tests/Xumm/XummTests.cs [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        var claims = await AuthenticateUserAsync(server);
+
+        // Assert
+        AssertClaim(claims, claimType, claimValue);
+    }
+
+    [Theory]
+    [InlineData(false)]
+    [InlineData(true)]
+    public async Task BuildChallengeUrl_Generates_Correct_Url(bool usePkce)
```
Comment (martincostello) [inline test/AspNet.Security.OAuth.Providers.Tests/Xumm/XummTests.cs]: This test can be removed as your handler doesn't override `ExchangeCodeAsync()`, so it's just testing the base class.

**Review by martincostello — COMMENTED** [review]
Thanks for your contribution! Just two small comments.

File: test/AspNet.Security.OAuth.Providers.Tests/Xumm/XummTests.cs (hunk 2 above)
Comment (DominiqueBlomsma) [reply test/AspNet.Security.OAuth.Providers.Tests/Xumm/XummTests.cs]: Updated to the recommended version.

Comment (DominiqueBlomsma) [reply test/AspNet.Security.OAuth.Providers.Tests/Xumm/XummTests.cs]: You're right, the redundant test has been removed.

**Comment by DominiqueBlomsma** [comment]
> Thanks for your contribution! Just two small comments.

My pleasure and resolved the comments.

**Review by martincostello — APPROVED** [review]

**Comment by martincostello** [comment]
Thanks again - this package is now available from [NuGet.org](https://www.nuget.org/packages/AspNet.Security.OAuth.Xumm/).

**Pull request merged**